locales/tr-TR.json

```diff
@@ -0,0 +1,10 @@
+[
+  "az önce",
+  ["%s saniye önce", "%s saniye önce"],
```

Comment:
these two looks like the same thing, so it can be a string instead of array here 😅